peps/pep-0777.rst

```diff
@@ -7,7 +7,7 @@ Status: Draft
 Type: Standards Track
 Topic: Packaging
 Created: 09-Oct-2024
-Post-History:
+Post-History: `10-Oct-2024 <https://discuss.python.org/t/pep-777-how-to-re-invent-the-wheel/67484>`_
```

Comment:
Interesting.  I didn't realize PEP 1 had been updated to include a link in the `Post-History`.  At the least you should probably also add a `Discussions-To` header.